docs/Product-Guides/Benefits-API.md

```diff
@@ -1,40 +1,253 @@
 # Integrating the Benefits API
 
-Once an employer has connected their system to your application, you can use Finch's [Benefits API](https://developer.tryfinch.com/docs/reference/b3A6MTg4Mzc2MTk-create-benefit) to manage benefits (401k, HSA, Loans, etc).
+Finch's [Benefits API](https://developer.tryfinch.com/docs/reference/b3A6MTg4Mzc2MTk-create-benefit) endpoints allow users to read and write benefits information (401k, Health Savings Accounts, Loans, etc) across providers using a single, unified API.
 
-Working with benefits through the Finch API is a simple two-phase process:
-1. Benefits are configured at the company level. When a new benefit is created or an existing benefit is read from the system, you will receive a `benefit_id` for it.
-2. You can use the `benefit_id` to manage the enrollment of employees in that benefit.
+## How do payroll benefit deductions work?
 
-----
+Since benefits work very differently across different providers, it is important to know a little about how payroll and benefits work in general.
+
+Each payroll contains four important dates to know.
+
+1. Payroll Start Date - The first day of the pay period
+2. Payroll End Date - The last day of the pay period
+3. Payroll Close Date - The last date to make changes for that pay period
+4. Paycheck Date - The date on which employees are paid
+
+When considering benefit deductions, it is important to submit any employee deductions before the pay close date in order to affect for the current pay period. If any deductions are submitted after the payroll close date, they will only affect the next pay period, not the current. Since each payroll close date is different per provider, it is important to know this date and set proper expectations with your customers.
```

Comment:
to include deductions and contributions
```suggestion
When considering benefits, it is important to submit any employee benefit changes before the pay close date in order to affect for the current pay period. If any changes are submitted after the payroll close date, they will only affect the next pay period, not the current. Since each payroll close date is different per provider, it is important to know this date and set proper expectations with your customers.
```